packages/web-react/src/components/FileUploader/useFileUploaderInput.ts

```diff
@@ -0,0 +1,162 @@
+import { ChangeEvent, DragEvent } from 'react';
+import { useDragAndDrop } from '../../hooks';
+import { useFileUploaderContext } from './FileUploaderContext';
+
+export interface UseFileUploaderInputProps {
+  accept?: string;
+  isMultiple?: boolean;
+  maxFileSize: number;
+  maxUploadedFiles: number;
+  onError?: (error: string) => void;
+}
+
+export interface UseFileUploaderInputReturn {
```

Comment:
Can we extend `DragAndDropHandlingProps` (https://github.com/lmc-eu/spirit-design-system/pull/804/files#diff-06c03e5ece10932ff2744e5f2fb91da48e7ae87640d7f713d7d3b3115167d58dR3) or prepare some base interface for drag and drop that are these two interfaces using, please?

Reply:
Done